grails-app/controllers/org/pih/warehouse/requisition/RequisitionTemplateController.groovy

```diff
@@ -86,6 +86,79 @@ class RequisitionTemplateController {
         }
     }
 
+    def exportPage = {
```

Comment:
Name this closure `export` instead of `exportPage`. And you don't need the render on the next line. Just leave the closure empty and rename the GSP export.gsp

```def export = { }```

Reply:
There is already closure called export in this controller. It is used for exporting items in stocklist.